### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by nkokina**

### CODE REVIEW

File: DropdownTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        WebElement dropdownElement = driver.findElement(By.id("dropdown"));
+        Select select = new Select(dropdownElement);
+        List<WebElement> options = select.getOptions();
+        List<String> expectedOptions = new ArrayList<>();
+        expectedOptions.add("Please select an option");
+        expectedOptions.add("Option 1");
+        expectedOptions.add("Option 2");
+        List<String> actualOptions = options.stream().map(option -> option.getText()).toList();
+        Assert.assertEquals(expectedOptions, actualOptions, "Not all dropdown elements");
+        select.selectByValue("1");
```
Comment (voropa) [inline DropdownTest.java:25]: why you select the same option twice?

File: DropdownTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        select.selectByIndex(0);
+        String expectedOptionOne = "Option 1";
+        String actualOptionOne = select.getFirstSelectedOption().getText();
+        Assert.assertEquals(expectedOptionOne, actualOptionOne, "Option 1 didn't get out");
+        select.selectByValue("2");
```
Comment (voropa) [inline DropdownTest.java:29]: Same here

File: InputsTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import org.testng.Assert;
+
+import java.util.List;
+
+public class InputsTest extends BaseTest {
+
+    @Test
+    public void selectInputsTest() {
+        driver.get("http://the-internet.herokuapp.com/inputs");
+        List<WebElement> inputs = driver.findElements(By.tagName("input"));
```
Comment (voropa) [inline InputsTest.java]: There's a single input on the page, why you find it as List<WebElement>?

**Review by voropa — CHANGES_REQUESTED** [review]

File: DropdownTest.java (hunk 2 above)
Comment (nkokina) [reply DropdownTest.java:29]: Test corrected

File: InputsTest.java (hunk 3 above)
Comment (nkokina) [reply InputsTest.java]: Test corrected

**Review by voropa — APPROVED** [review]

**Pull request merged**